docs/crictl.md

```diff
@@ -75,6 +75,16 @@ The endpoint can be set in three ways:
 - By setting environment variables `CONTAINER_RUNTIME_ENDPOINT` and `IMAGE_SERVICE_ENDPOINT`
 - By setting the endpoint in the config file `--config=/etc/crictl.yaml`
 
+If the endpoint is not set then it works as follows :
```

Comment:
suggest `If the runtime endpoint is not set crictl will, by default, try to connect using`

Reply:
Sounds better. Updated